Sources/Client/Client/ClientLogger.swift

```diff
@@ -341,13 +354,13 @@ public final class ClientLogger {
 extension Date {
     /// A string of the date for the `ClientLogger`.
     public var log: String {
-        return DateFormatter.log?.string(from: self) ?? ""
+        return DateFormatter.log.string(from: self)
     }
 }
 
 extension DateFormatter {
     /// A date formatter for `ClientLogger`.
-    public static var log: DateFormatter? = {
```

Comment:
It was useful to remove noise from dates.

Reply:
I'll add as an option